compiler_tester/src/lib.rs

```diff
@@ -143,12 +148,17 @@ impl CompilerTester {
         let _: Vec<()> = tests
             .into_par_iter()
             .map(|(test, compiler, mode)| {
+                let mode_string = mode.clone().to_string();
+                let specialized_debug_config = self
+                    .debug_config
+                    .clone()
```

Comment:
`.to_owned()` is more universal if you an owned object from a reference.
But probably `.as_ref()` would suffice here.